### PR TITLE
EOS-24934: package validation restored in post_install phases

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
Signed-off-by: Rahul Tripathi <rahul.tripathi@seagate.com>

# Problem Statement
- Python packages should be validated in post_install

# Design
-  The validation of correct python packages should be restored in both, utils rpm post_install as well as utils mini-prov post_install phase

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: Y
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: Y
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: jenkins/build.sh [hunk 1]
```diff
@@ -63,35 +63,35 @@ sed -i -e "s|<INSTALL_PATH>|${INSTALL_PATH}|g" utils-post-install
...
+ echo "PACKAGE_LIST=\""  >> utils-pre-install
+ /bin/cat python_requirements.txt >> utils-pre-install
+ if [ -f "python_requirements.ext.txt" ]; then
+     /bin/cat python_requirements.ext.txt >> utils-pre-install
+ fi
+ echo "\""  >> utils-pre-install
+ echo "rc=0
+ for package in \$PACKAGE_LIST
+ do
+     pip3 freeze | grep \$package > /dev/null
```
Comment (sachinpunadikar) [inline jenkins/build.sh:77]: Per my observation, this command needs an update. Refer to the code from main branch

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on the comment

File: jenkins/build.sh (hunk 1 above)
Comment (rahul-tripathi-git) [reply jenkins/build.sh]: Updated.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by suryakumar1024 — APPROVED** [review]

**Review by vastradparayya — APPROVED** [review]

**Review by veerendra-simha-garikipati — APPROVED** [review]

**Pull request merged**